### PR TITLE
Lynch Testing - Additional Review

### PR DESCRIPTION
**Pull request by jy8474np**
I feel like my tests look good, but there are a few squigglies under a couple of my variable names. Take a look and let me know any thoughts. Thanks!

### CODE REVIEW

File: image_display_save.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from PIL import Image
+import shutil
+import requests
+import os
+import glob
+from pprint import pprint
+
+cover = Image.open("/Users/mtlynchjr/Desktop/Final Semester/pillow_test_01/env/golden_gate.jpg") # Existing image opened from directory folder
```
Comment (yp0975eu) [inline image_display_save.py:8]: This isn't going to work. The hard coded path needs to be removed.

File: test02_cover_art.py [hunk 2]
```diff
@@ -8,23 +8,34 @@
...
+                test_image_url = i.get("#text")
+                return test_image_url
+        expected_url = cover_art_api.get_image_by_size("images" , "size")
+        self.assertEqual(test_image_url , expected_url)
 
-        pprint(cover_art_api.cover_response)
-        pprint(self.test_cover_response)
+    def test_image_download(self):
+        test_url = cover_art_api.url
+        r = requests.get(test_url)
```
Comment (yp0975eu) [inline test02_cover_art.py:32]: This can be one line 

        r = requests.get( cover_art_api.url)

File: test02_cover_art.py [hunk 3]
```diff
@@ -8,23 +8,34 @@
...
-        test_cover_response = requests.get(cover_art_api.url, params=test_query).json()
-        self.assertEqual(cover_art_api.cover_response, self.test_cover_response)
-    
-    def test_image_download(self):
-        self.images = self.test_cover_response.get('album').get('image')
-        self.assertEqual(self.test_cover_response, cover_art_api.cover_response)
+        test_query_results = cover_art_api.get_album_art(test_album, test_artist)
+        self.assertEqual(test_query , test_query_results)
+
+    def test_get_image_by_size(self):
```
Comment (yp0975eu) [inline test02_cover_art.py:19]: What is this testing? You have both the function call `cover_art_api.get_image_by_size("images" , "size")` and the implementation, lines 23 - 26 in the same test.

**Review by yp0975eu — CHANGES_REQUESTED** [review]
This needs work before merging.  If you merge this to master as it is, it will break.

**Comment by jy8474np** [comment]
Test pages merged and re-written.

**Pull request closed without merging**